engines/agi/saveload.cpp

```diff
@@ -927,6 +935,11 @@ bool AgiEngine::getSavegameInformation(int16 slotId, Common::String &saveDescrip
 		saveDescription += saveGameDescription;
 		saveIsValid = true;
 
+		if (_game._vm->getLanguage() == Common::HE_ISR) {
+			Common::U32String nameU32String = saveDescription.decode(Common::kUtf8);
+			saveDescription = nameU32String.encode(Common::kWindows1255);
```

Comment:
Can be simplified: `saveDescription = saveDescription.decode().encode(Common::kWindows1255);`

Reply:
Agreed.